src/js/providers/default.ts

```diff
@@ -78,6 +78,12 @@ export type ProviderEvents = {
     [Event.MEDIA_SEEK]: {
         position: number;
         offset: number;
+        duration: number;
+        currentTime: number;
+        seekRange: SeekRange;
+        metadata: {
+            currentTime: number;
+        };
```

Comment:
Included this `metadata` property to mirror most of the properties in the `MEDIA_TIME` event, but can remove if not needed.